x-pack/legacy/plugins/snapshot_restore/public/app/components/policy_form/steps/step_retention.tsx

```diff
@@ -93,13 +93,19 @@ export const PolicyStepRetention: React.FunctionComponent<StepProps> = ({
             <EuiFieldNumber
               value={retention.expireAfterValue || ''}
               onBlur={() => setTouched({ ...touched, expireAfterValue: true })}
+              onKeyDown={e => {
```

Comment:
Rather than preventing the user from entering the `-` symbol, I think it might be better to add a new validation rule that prevents negative numbers. This way, a validation error message will appear as well, and the user will not be able to proceed to the next step until it is corrected.

For example, there is currently validation in place so that a user cannot set the `min_count` greater than `max_count`.

<img width="1288" alt="Screen Shot 2019-11-25 at 1 47 48 PM" src="https://user-images.githubusercontent.com/5226211/69568789-4a9e1900-0f8a-11ea-95cc-6414c9a5cc85.png">

You should be able to modify this file: https://github.com/elastic/kibana/blob/cda7eba4c9dbb3739c726a194ceed155111f15fc/x-pack/legacy/plugins/snapshot_restore/public/app/services/validation/validate_policy.ts

Reply:
Hi @alisonelizabeth thanks for the tip. It surely looks better as a validation.
I did some testing and it looks like the image below.

I also tryied to add a Jest test, but I don't know eactly how to run it locally (do you? I mean, doyou know how I could have run specifically the `x-pack/legacy/plugins/snapshot_restore/__jest__/client_integration/policy_add.test.ts` test locally before pushing?)

Let me know what do you think!

![Bildschirmfoto vom 2019-11-25 20-17-19](https://user-images.githubusercontent.com/23581855/69586752-57d6fa00-0fc2-11ea-8ef1-42eec280c9a1.png)